bigtable/admin.go

```diff
@@ -1663,3 +1664,53 @@ func (ac *AdminClient) UpdateBackup(ctx context.Context, cluster, backup string,
 	_, err = ac.tClient.UpdateBackup(ctx, req)
 	return err
 }
+
+// GetIamPolicy gets the IAM access control policy for the specified backup.
+func (ac *AdminClient) GetIamPolicy(ctx context.Context, cluster, backup string) (*iampb.Policy, error) {
+	ctx = mergeOutgoingMetadata(ctx, ac.md)
+	prefix := ac.instancePrefix()
+	backupPath := prefix + "/clusters/" + cluster + "/backups/" + backup
+
+	req := iampb.GetIamPolicyRequest{Resource: backupPath}
```

Comment:
Looks like this function should return an `*iam.Handle` and use https://pkg.go.dev/cloud.google.com/go/iam#InternalNewHandleClient to do it, rather than making the request directly.